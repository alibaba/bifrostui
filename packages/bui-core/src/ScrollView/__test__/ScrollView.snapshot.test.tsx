import { snapshotTest } from 'testing';

describe('ScrollView snapshot', () => {
  snapshotTest('ScrollView');
});
