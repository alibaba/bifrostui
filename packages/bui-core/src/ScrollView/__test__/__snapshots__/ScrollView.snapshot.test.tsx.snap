// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollView snapshot ScrollView demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-scroll bui-scroll-view-scroll-y"
    onScroll={[Function]}
    onTouchMove={[Function]}
    style={
      Object {
        "height": "400px",
        "width": "100%",
      }
    }
  >
    <div
      className="item"
    >
      0
    </div>
    <div
      className="item"
    >
      1
    </div>
    <div
      className="item"
    >
      2
    </div>
    <div
      className="item"
    >
      3
    </div>
    <div
      className="item"
    >
      4
    </div>
    <div
      className="item"
    >
      5
    </div>
    <div
      className="item"
    >
      6
    </div>
    <div
      className="item"
    >
      7
    </div>
    <div
      className="item"
    >
      8
    </div>
    <div
      className="item"
    >
      9
    </div>
    <div
      className="item"
    >
      10
    </div>
    <div
      className="item"
    >
      11
    </div>
    <div
      className="item"
    >
      12
    </div>
    <div
      className="item"
    >
      13
    </div>
    <div
      className="item"
    >
      14
    </div>
    <div
      className="item"
    >
      15
    </div>
    <div
      className="item"
    >
      16
    </div>
    <div
      className="item"
    >
      17
    </div>
    <div
      className="item"
    >
      18
    </div>
    <div
      className="item"
    >
      19
    </div>
    <div
      className="item"
    >
      20
    </div>
    <div
      className="item"
    >
      21
    </div>
    <div
      className="item"
    >
      22
    </div>
    <div
      className="item"
    >
      23
    </div>
    <div
      className="item"
    >
      24
    </div>
    <div
      className="item"
    >
      25
    </div>
    <div
      className="item"
    >
      26
    </div>
    <div
      className="item"
    >
      27
    </div>
    <div
      className="item"
    >
      28
    </div>
    <div
      className="item"
    >
      29
    </div>
    <div
      className="item"
    >
      30
    </div>
    <div
      className="item"
    >
      31
    </div>
    <div
      className="item"
    >
      32
    </div>
    <div
      className="item"
    >
      33
    </div>
    <div
      className="item"
    >
      34
    </div>
    <div
      className="item"
    >
      35
    </div>
    <div
      className="item"
    >
      36
    </div>
    <div
      className="item"
    >
      37
    </div>
    <div
      className="item"
    >
      38
    </div>
    <div
      className="item"
    >
      39
    </div>
    <div
      className="item"
    >
      40
    </div>
    <div
      className="item"
    >
      41
    </div>
    <div
      className="item"
    >
      42
    </div>
    <div
      className="item"
    >
      43
    </div>
    <div
      className="item"
    >
      44
    </div>
    <div
      className="item"
    >
      45
    </div>
    <div
      className="item"
    >
      46
    </div>
    <div
      className="item"
    >
      47
    </div>
    <div
      className="item"
    >
      48
    </div>
    <div
      className="item"
    >
      49
    </div>
    <div
      className="item"
    >
      50
    </div>
    <div
      className="item"
    >
      51
    </div>
    <div
      className="item"
    >
      52
    </div>
    <div
      className="item"
    >
      53
    </div>
    <div
      className="item"
    >
      54
    </div>
    <div
      className="item"
    >
      55
    </div>
    <div
      className="item"
    >
      56
    </div>
    <div
      className="item"
    >
      57
    </div>
    <div
      className="item"
    >
      58
    </div>
    <div
      className="item"
    >
      59
    </div>
    <div
      className="item"
    >
      60
    </div>
    <div
      className="item"
    >
      61
    </div>
    <div
      className="item"
    >
      62
    </div>
    <div
      className="item"
    >
      63
    </div>
    <div
      className="item"
    >
      64
    </div>
    <div
      className="item"
    >
      65
    </div>
    <div
      className="item"
    >
      66
    </div>
    <div
      className="item"
    >
      67
    </div>
    <div
      className="item"
    >
      68
    </div>
    <div
      className="item"
    >
      69
    </div>
    <div
      className="item"
    >
      70
    </div>
    <div
      className="item"
    >
      71
    </div>
    <div
      className="item"
    >
      72
    </div>
    <div
      className="item"
    >
      73
    </div>
    <div
      className="item"
    >
      74
    </div>
    <div
      className="item"
    >
      75
    </div>
    <div
      className="item"
    >
      76
    </div>
    <div
      className="item"
    >
      77
    </div>
    <div
      className="item"
    >
      78
    </div>
    <div
      className="item"
    >
      79
    </div>
    <div
      className="item"
    >
      80
    </div>
    <div
      className="item"
    >
      81
    </div>
    <div
      className="item"
    >
      82
    </div>
    <div
      className="item"
    >
      83
    </div>
    <div
      className="item"
    >
      84
    </div>
    <div
      className="item"
    >
      85
    </div>
    <div
      className="item"
    >
      86
    </div>
    <div
      className="item"
    >
      87
    </div>
    <div
      className="item"
    >
      88
    </div>
    <div
      className="item"
    >
      89
    </div>
    <div
      className="item"
    >
      90
    </div>
    <div
      className="item"
    >
      91
    </div>
    <div
      className="item"
    >
      92
    </div>
    <div
      className="item"
    >
      93
    </div>
    <div
      className="item"
    >
      94
    </div>
    <div
      className="item"
    >
      95
    </div>
    <div
      className="item"
    >
      96
    </div>
    <div
      className="item"
    >
      97
    </div>
    <div
      className="item"
    >
      98
    </div>
    <div
      className="item"
    >
      99
    </div>
  </div>
</div>
`;

exports[`ScrollView snapshot ScrollView demo snapshot 0 2`] = `
<div
  className="bui-scroll bui-scroll-view-scroll-x"
  onScroll={[Function]}
  onTouchMove={[Function]}
  style={
    Object {
      "width": "50vw",
    }
  }
>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "8px",
        "width": "max-content",
      }
    }
  >
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      0
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      1
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      2
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      3
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      4
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      5
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      6
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      7
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      8
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      9
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      10
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      11
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      12
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      13
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      14
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      15
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      16
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      17
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      18
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      19
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      20
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      21
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      22
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      23
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      24
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      25
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      26
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      27
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      28
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      29
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      30
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      31
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      32
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      33
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      34
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      35
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      36
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      37
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      38
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      39
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      40
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      41
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      42
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      43
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      44
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      45
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      46
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      47
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      48
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      49
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      50
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      51
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      52
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      53
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      54
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      55
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      56
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      57
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      58
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      59
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      60
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      61
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      62
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      63
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      64
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      65
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      66
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      67
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      68
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      69
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      70
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      71
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      72
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      73
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      74
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      75
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      76
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      77
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      78
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      79
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      80
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      81
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      82
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      83
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      84
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      85
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      86
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      87
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      88
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      89
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      90
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      91
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      92
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      93
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      94
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      95
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      96
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      97
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      98
    </div>
    <div
      className="item"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      99
    </div>
  </div>
</div>
`;

exports[`ScrollView snapshot ScrollView demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    回到顶部
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    滚动到200px
  </button>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    滚动到[50]文本处
  </button>
  <div
    className="bui-scroll bui-scroll-view-scroll-y"
    onScroll={[Function]}
    onTouchMove={[Function]}
    style={
      Object {
        "height": "400px",
        "width": "100%",
      }
    }
  >
    <div
      className="item"
      id="d0"
    >
      0
    </div>
    <div
      className="item"
      id="d1"
    >
      1
    </div>
    <div
      className="item"
      id="d2"
    >
      2
    </div>
    <div
      className="item"
      id="d3"
    >
      3
    </div>
    <div
      className="item"
      id="d4"
    >
      4
    </div>
    <div
      className="item"
      id="d5"
    >
      5
    </div>
    <div
      className="item"
      id="d6"
    >
      6
    </div>
    <div
      className="item"
      id="d7"
    >
      7
    </div>
    <div
      className="item"
      id="d8"
    >
      8
    </div>
    <div
      className="item"
      id="d9"
    >
      9
    </div>
    <div
      className="item"
      id="d10"
    >
      10
    </div>
    <div
      className="item"
      id="d11"
    >
      11
    </div>
    <div
      className="item"
      id="d12"
    >
      12
    </div>
    <div
      className="item"
      id="d13"
    >
      13
    </div>
    <div
      className="item"
      id="d14"
    >
      14
    </div>
    <div
      className="item"
      id="d15"
    >
      15
    </div>
    <div
      className="item"
      id="d16"
    >
      16
    </div>
    <div
      className="item"
      id="d17"
    >
      17
    </div>
    <div
      className="item"
      id="d18"
    >
      18
    </div>
    <div
      className="item"
      id="d19"
    >
      19
    </div>
    <div
      className="item"
      id="d20"
    >
      20
    </div>
    <div
      className="item"
      id="d21"
    >
      21
    </div>
    <div
      className="item"
      id="d22"
    >
      22
    </div>
    <div
      className="item"
      id="d23"
    >
      23
    </div>
    <div
      className="item"
      id="d24"
    >
      24
    </div>
    <div
      className="item"
      id="d25"
    >
      25
    </div>
    <div
      className="item"
      id="d26"
    >
      26
    </div>
    <div
      className="item"
      id="d27"
    >
      27
    </div>
    <div
      className="item"
      id="d28"
    >
      28
    </div>
    <div
      className="item"
      id="d29"
    >
      29
    </div>
    <div
      className="item"
      id="d30"
    >
      30
    </div>
    <div
      className="item"
      id="d31"
    >
      31
    </div>
    <div
      className="item"
      id="d32"
    >
      32
    </div>
    <div
      className="item"
      id="d33"
    >
      33
    </div>
    <div
      className="item"
      id="d34"
    >
      34
    </div>
    <div
      className="item"
      id="d35"
    >
      35
    </div>
    <div
      className="item"
      id="d36"
    >
      36
    </div>
    <div
      className="item"
      id="d37"
    >
      37
    </div>
    <div
      className="item"
      id="d38"
    >
      38
    </div>
    <div
      className="item"
      id="d39"
    >
      39
    </div>
    <div
      className="item"
      id="d40"
    >
      40
    </div>
    <div
      className="item"
      id="d41"
    >
      41
    </div>
    <div
      className="item"
      id="d42"
    >
      42
    </div>
    <div
      className="item"
      id="d43"
    >
      43
    </div>
    <div
      className="item"
      id="d44"
    >
      44
    </div>
    <div
      className="item"
      id="d45"
    >
      45
    </div>
    <div
      className="item"
      id="d46"
    >
      46
    </div>
    <div
      className="item"
      id="d47"
    >
      47
    </div>
    <div
      className="item"
      id="d48"
    >
      48
    </div>
    <div
      className="item"
      id="d49"
    >
      49
    </div>
    <div
      className="item"
      id="d50"
    >
      50
    </div>
    <div
      className="item"
      id="d51"
    >
      51
    </div>
    <div
      className="item"
      id="d52"
    >
      52
    </div>
    <div
      className="item"
      id="d53"
    >
      53
    </div>
    <div
      className="item"
      id="d54"
    >
      54
    </div>
    <div
      className="item"
      id="d55"
    >
      55
    </div>
    <div
      className="item"
      id="d56"
    >
      56
    </div>
    <div
      className="item"
      id="d57"
    >
      57
    </div>
    <div
      className="item"
      id="d58"
    >
      58
    </div>
    <div
      className="item"
      id="d59"
    >
      59
    </div>
    <div
      className="item"
      id="d60"
    >
      60
    </div>
    <div
      className="item"
      id="d61"
    >
      61
    </div>
    <div
      className="item"
      id="d62"
    >
      62
    </div>
    <div
      className="item"
      id="d63"
    >
      63
    </div>
    <div
      className="item"
      id="d64"
    >
      64
    </div>
    <div
      className="item"
      id="d65"
    >
      65
    </div>
    <div
      className="item"
      id="d66"
    >
      66
    </div>
    <div
      className="item"
      id="d67"
    >
      67
    </div>
    <div
      className="item"
      id="d68"
    >
      68
    </div>
    <div
      className="item"
      id="d69"
    >
      69
    </div>
    <div
      className="item"
      id="d70"
    >
      70
    </div>
    <div
      className="item"
      id="d71"
    >
      71
    </div>
    <div
      className="item"
      id="d72"
    >
      72
    </div>
    <div
      className="item"
      id="d73"
    >
      73
    </div>
    <div
      className="item"
      id="d74"
    >
      74
    </div>
    <div
      className="item"
      id="d75"
    >
      75
    </div>
    <div
      className="item"
      id="d76"
    >
      76
    </div>
    <div
      className="item"
      id="d77"
    >
      77
    </div>
    <div
      className="item"
      id="d78"
    >
      78
    </div>
    <div
      className="item"
      id="d79"
    >
      79
    </div>
    <div
      className="item"
      id="d80"
    >
      80
    </div>
    <div
      className="item"
      id="d81"
    >
      81
    </div>
    <div
      className="item"
      id="d82"
    >
      82
    </div>
    <div
      className="item"
      id="d83"
    >
      83
    </div>
    <div
      className="item"
      id="d84"
    >
      84
    </div>
    <div
      className="item"
      id="d85"
    >
      85
    </div>
    <div
      className="item"
      id="d86"
    >
      86
    </div>
    <div
      className="item"
      id="d87"
    >
      87
    </div>
    <div
      className="item"
      id="d88"
    >
      88
    </div>
    <div
      className="item"
      id="d89"
    >
      89
    </div>
    <div
      className="item"
      id="d90"
    >
      90
    </div>
    <div
      className="item"
      id="d91"
    >
      91
    </div>
    <div
      className="item"
      id="d92"
    >
      92
    </div>
    <div
      className="item"
      id="d93"
    >
      93
    </div>
    <div
      className="item"
      id="d94"
    >
      94
    </div>
    <div
      className="item"
      id="d95"
    >
      95
    </div>
    <div
      className="item"
      id="d96"
    >
      96
    </div>
    <div
      className="item"
      id="d97"
    >
      97
    </div>
    <div
      className="item"
      id="d98"
    >
      98
    </div>
    <div
      className="item"
      id="d99"
    >
      99
    </div>
  </div>
</div>
`;
